src/visitor.rs

```diff
@@ -229,6 +229,7 @@ impl<'a> FmtVisitor<'a> {
     fn visit_fn(
         &mut self,
         fk: visit::FnKind,
+        generics: &ast::Generics,
```

Comment:
Notice that this parameter isn't used in the first variant of the match statement below because it is redundant. (More on that in my other PR comment below)